style.css

```diff
@@ -0,0 +1,1297 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    overflow-x: hidden;
```

Comment:
This hidden property is affecting your design on medium and small screens.

![hidden](https://user-images.githubusercontent.com/31304943/69759564-b935cf00-1162-11ea-95d1-faa802883860.png)